### PR TITLE
PAT-1320 create api for nomis court appearances

### PR DESCRIPTION
**Pull request by steverendell**

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/CourtEvent.java [hunk 1]
```diff
@@ -79,10 +79,10 @@
     @ManyToOne
     @NotFound(action = IGNORE)
     @JoinColumnsOrFormulas(value = {
-            @JoinColumnOrFormula(formula = @JoinFormula(value = "'" + EVENT_TYPE + "'", referencedColumnName = "domain")),
+            @JoinColumnOrFormula(formula = @JoinFormula(value = "'" + REASON + "'", referencedColumnName = "domain")),
```
Comment (steverendell) [inline src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/CourtEvent.java:82]: it turns out the codes for MOVE_RSN where pareet is CRN are a perfect match for values of EVENT_SUB_TYPE in T3

File: src/main/java/uk/gov/justice/hmpps/prison/api/resource/MovementResource.java [hunk 2]
```diff
@@ -13,16 +13,7 @@
...
-import uk.gov.justice.hmpps.prison.api.model.Movement;
-import uk.gov.justice.hmpps.prison.api.model.MovementCount;
-import uk.gov.justice.hmpps.prison.api.model.OffenderIn;
-import uk.gov.justice.hmpps.prison.api.model.OffenderInReception;
-import uk.gov.justice.hmpps.prison.api.model.OffenderMovement;
-import uk.gov.justice.hmpps.prison.api.model.OffenderOut;
-import uk.gov.justice.hmpps.prison.api.model.OffenderOutTodayDto;
-import uk.gov.justice.hmpps.prison.api.model.RollCount;
-import uk.gov.justice.hmpps.prison.api.model.TransferSummary;
+import uk.gov.justice.hmpps.prison.api.model.*;
```
Comment (matt-crowson-moj) [inline src/main/java/uk/gov/justice/hmpps/prison/api/resource/MovementResource.java:16]: Minor point but should not be using wildcard imports (unless it is a unit test)....Intellij setting changes will resolve this (assuming you use this).

**Review by matt-crowson-moj — APPROVED** [review]
This looks OK to me.  You probably want someone closer to the change though to check also.

File: src/main/java/uk/gov/justice/hmpps/prison/api/resource/MovementResource.java (hunk 2 above)
Comment (steverendell) [reply src/main/java/uk/gov/justice/hmpps/prison/api/resource/MovementResource.java]: sorted

**Review by connormaglynn — APPROVED** [review]
LGTM

**Pull request merged**